fastlane/Fastfile

```diff
@@ -228,6 +228,18 @@ end
 
 desc "Verifies all tests pass and the current state of the repo is valid"
 private_lane :validate_repo do
+  # Ensure that no ENV vars which interefere with testing are set
```

Comment:
I suppose? I guess I look at my normal testing process to be more fluid. Maybe I'm setting ENV variables to explore things... In comparison, the release process should be pretty locked down.

Does that make sense? If not, do you have a sense of where you would put it?